README.md

```diff
@@ -15,20 +15,28 @@ The application requires the Firebase to run. The debug build connects to the lo
    firebase emulators:start
    ```
 
-**To disable the emulator in the dev environment, disable the line of codes in `lib/main.dart` or build in production mode**
+### Connect to a Android Device
+
+1. Make sure firebase emulators is running.
+
+2. Connect your device via USB and enable developer mode.
+
+3. Head over to your Android SDK platform-tools folder `android_sdk/platform-tools/`
+
+4. Connect your Android device with Firebase Emulator Servers:
+
+    Firestore
+    ```bash
+    adb reverse tcp:8080 tcp:8080
+    ```
+    Storage
+    ```bash
+    adb reverse tcp:9199 tcp:9199
+    ```
+5. Run the app from your IDE.
 
-```dart
-  if (kDebugMode) {
-    try {
-      FirebaseFirestore.instance.useFirestoreEmulator('localhost', 8080);
-    } catch (e) {
-      // ignore: avoid_print
-      print(e);
-    }
-  }
-```
 
-### How to initialize Firebase
+### How to initialize Firebase Production
```

Comment:
I'm not sure if it's instruction for production. It's just a general instruction.